holoviews/plotting/plotly/raster.py

```diff
@@ -99,6 +109,9 @@ def get_data(self, element, ranges, style):
 
 class QuadMeshPlot(RasterPlot):
 
+    nodata = param.Integer(default=None, doc="""
+        Missing data (NaN) value for integer data""")
```

Comment:
```suggestion
        Optional missing-data value for integer data.
        If non-None, data with this value will be replaced with NaN so that it is transparent when plotted.""")
```